entity-framework/core/performance/advanced-performance-topics.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Advanced Performance Topics
+description: Advanced performance topics for Entity Framework Core
+author: rick-anderson
+ms.author: riande
+ms.date: 12/9/2020
+uid: core/performance/advanced-performance-topics
+---
+# Advanced Performance Topics
+
+## DbContext pooling
+
+`AddDbContextPool` enables pooling of `DbContext` instances. Context pooling can increase throughput in high-scale scenarios such as web servers by reusing context instances, rather than creating new instances for each request.
```

Comment:
Maybe move the last sentence in this topic up here?

Reply:
Are you referring to the paragraph that starts with "Context pooling is intended for scenarios where the context configuration, which includes services resolved, is fixed between requests."? It seems OK to me under the limitations, what do you have in mind?